stdlib/typing_extensions.pyi

```diff
@@ -155,6 +156,13 @@ def get_type_hints(
     include_extras: bool = False,
 ) -> dict[str, Any]: ...
 def get_args(tp: Any) -> tuple[Any, ...]: ...
+@overload
+def get_origin(tp: GenericAlias) -> type: ...
+@overload
+def get_origin(tp: ParamSpecArgs | ParamSpecKwargs) -> ParamSpec: ...
+@overload
+def get_origin(tp: UnionType) -> type[UnionType]: ...
+@overload
 def get_origin(tp: Any) -> Any | None: ...
```

Comment:
You should be able to fix the CI errors by doing version-dependent overloads like this:

```suggestion
if sys.version_info >= (3, 10):
    @overload
    def get_origin(tp: ParamSpecArgs | ParamSpecKwargs) -> ParamSpec: ...
    @overload
    def get_origin(tp: UnionType) -> type[UnionType]: ...
if sys.version_info >= (3, 9):
    @overload
    def get_origin(tp: GenericAlias) -> type: ...
    @overload
    def get_origin(tp: Any) -> Any | None: ...
else:
    def get_origin(tp: Any) -> Any | None: ...
```

Reply:
Will fix CI right now